app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -76,6 +76,86 @@ def bulk_search
         render :json => ret
       end
 
+      api :PUT, '/tasks/bulk_resume', N_('Resume all paused error tasks')
+      param :search, String, :desc => N_('Resume tasks matching search string')
+      param :task_ids, Array, :desc => N_('Resume specific tasks by id')
+      def bulk_resume
+        scope = resource_scope
+        scope = scope.search_for(params[:search]) if params[:search]
+        scope = scope.select('DISTINCT foreman_tasks_tasks.*')
+        scope = scope.where(:state => :paused)
+        if params[:search].nil? && params[:task_ids].nil?
```

Comment:
I wonder if the `:state => :paused` should be also just a default scope, overriden by the :search/:task_ids option: reason: when you force unlock some task, it's switched into 'stopped' state, but you can still resume it.